File: 1-two-sum/two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        dicn={}
        for i,n in enumerate(nums):
            diff=target-n
            if n in dicn:
                return [dicn[n],i]
            else:
                dicn[diff]=i

        

File: 1-two-sum/two-sum.dart
class Solution {
  List<int> twoSum(List<int> nums, int target){
    for (var i = 0; i < nums.length; i += 1) {
      for (var j = i + 1; j < nums.length; j += 1) {
        if (nums[i] + nums[j] == target) {
          return [i, j];
        }
      }
    }
    return []; 
  }
}